masterlist.yaml

```diff
@@ -21792,7 +21792,19 @@ plugins:
     url:
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/86865/'
         name: 'HS Wilderness - Anise''s Cabin'
-  - name: 'HS(DragonBridge|Markarth|PlayerHomes|Riverwood|Solitude|Whiterun|Wilderness|Winterhold) - (Four Shields Tavern|Silver-Blood Inn|Alvor and Sigrid''s House|Faendal''s House|Hod and Gerdur''s House|Riverwood Trader|Sleeping Giant Inn|Sven and Hilde''s House|Temple of the Divines|The Winking Skeever|Arcadia''s Cauldron|Jorrvaskr|The Drunken Huntsman|Anise''s Cabin|Breezehome|Honeyside|Proudspire Manor|SeverinManor|Vlindrel Hall)\.esp'
+  - name: 'HSWhiterun - Belethor''s General Goods.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/87317'
+        name: 'HS Whiterun - Belethor''s General Goods'
+  - name: 'HSWhiterun - Dragonsreach.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/88753'
+        name: 'HS Whiterun - Dragonsreach'
+  - name: 'HSWhiterun - Ysolda''s House.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/89227'
+        name: 'HS Whiterun - Ysolda''s House'
```

Comment:
Could you please sort these alphabetically with the rest of your plugins & add trailing slashes to the URLs?